oaknational/Oak-Web-Application
# Feat/responsive style props

## Description

- Adds `responsive` helper which enables easy creation of responsive style props
- Creates responsive spacing props (margin and padding)
- Creates responsive background props
- Creates `Base > Flex` component (WIP) which uses aforementioned background and spacing props

## Issue(s)

Relates #38

## How to test

1. Go to {storybook}
2. Click on Flex
3. You should see:

![image](https://user-images.githubusercontent.com/12934669/169148742-3a646273-6790-44dd-afe4-269cbb1074ee.png)
